### DISCUSSION TITLE
Request support for updating "sits" and "torch" packages in MPC R environments

### DISCUSSION THREAD

**Discussion by gilbertocamara** [microsoft/PlanetaryComputer#189] [General]
Dear @TomAugspurger 

Could you please support the installation of the latest CRAN versions of "sits" and "torch" packages in the MPC R environment in the Hub? Some time ago, "sits" version 1.2.0 was available, as well as "torch". Today we are back to "sits" version 0.16.2 and missing "torch". Your help will be greatly appreciated.

**Comment by TomAugspurger** [comment]
My apologies @gilbertocamara. It seems my changes updating the image were accidentally reverted. That's been redeployed now and shouldn't happen again.

**Comment by gilbertocamara** [comment]
Wow!! Kudos for such a quick response!!!
